CHANGELOG.md

```diff
@@ -7,21 +7,21 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
-### Changed
 
+## [v1.0.0-alpha.6] - 2021-11-19
+
+*** This is (also) an alpha release with breaking changes (sorry) ***
+
+### Changed
 - Use `u8` as default SPI as Serial Word type
+- The Minimum Supported Rust Version (MSRV) is now 1.46.0
 
 ### Added
 - Added `Can` Controller Area Network traits.
 - `Error` traits for SPI, I2C and Serial traits. The error types used in those must
   implement these `Error` traits, which implies providing a conversion to a common
   set of error kinds. Generic drivers using these interfaces can then convert the errors
   to this common set to act upon them.
-- The `NoAcknowledgeAddress` and `NoAcknowledgeData` variants of the I2C `Error`
-  trait have been merged into `NoAcknowledge` with a `NoAcknowledgeSource` field
-  to differentiate between the two events. An additional `Unknown` variant is
-  provided in `NoAcknowledgeSource` for devices that can't differentiate between
-  the two events.
```

Comment:
Is the removal of this whole block intended?

Reply:
Yes, since this is the first release with the error types, I figured there is no need to document changes on how the current state came to be.